eclipse-theia/theia
# yarn upgrade + dep fix

#### What it does


The pull-request includes the following changes:
- [x] performs a `yarn upgrade` on the repo, consequently resolving known security vulnerabilities from dependencies.
- [x] sets `@electron/remote` range to avoid version `2.0.4` as the typings are bogus (fixed upstream but not yet in a patch release).

#### How to test


- confirm that the repo builds, and the application starts normally
- confirm that the tests pass
- confirm no `simple-get` vulnerability exists (ex: yarn audit | grep "simple-get")

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
